maths/percent.py

```diff
@@ -0,0 +1,19 @@
+"""
+Find the Percentage of two numbers
+wikipdia
+https://en.wikipedia.org/wiki/Percentage
+>>> percent(30.0,75.0)
+    40.0
+"""
+
+
+def percent(first_num: float, second_num: float) -> float:
```

Comment:
```suggestion
def percent(first_num: float | int, second_num: float| int) -> float:
```
Otherwise you can only pass `float`